core/src/test/java/cucumber/runtime/io/URLOutputStreamTest.java

```diff
@@ -4,40 +4,36 @@
 import gherkin.util.FixJava;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
-import org.webbitserver.HttpControl;
-import org.webbitserver.HttpHandler;
-import org.webbitserver.HttpRequest;
-import org.webbitserver.HttpResponse;
-import org.webbitserver.WebServer;
+import org.junit.rules.TemporaryFolder;
+import org.webbitserver.*;
```

Comment:
Ooops, it wasn't me, it was Intellij Idea. Sorry. Fixed in commits 6ad8186, cf811be